pandas/core/generic.py

```diff
@@ -4275,7 +4275,29 @@ def get_ftype_counts(self):
 
     @property
     def dtypes(self):
-        """Return the dtypes in this object."""
+        """
+        Return the dtypes in this object.
+
+        Notes
```

Comment:
I think you can remove the "Notes" header, and just make this the extended summary.